client/src/app.tsx

```diff
@@ -7,8 +7,7 @@ import { Document } from "./document";
 import Footer from "./footer";
 import Header from "./header";
 import { NoMatch } from "./routing";
-
-const ActiveBanner = lazy(() => import("./banners/active-banner"));
+import { MaybeBanner } from "./banners/maybe-banner";
```

Comment:
😸

Reply:
Not sure what the cat emoji means. Do you think the name is too whimsical?
I like a good bikeshed because I think names are important.
The intention by calling it "maybe" is that it's like a launchpad where rockets take off from. 
You definitely import it so it definitely becomes part of the main runtime bundle. But within, it *might* inject a banner. Maybe. Possibly. Potentially. Occasionally. 

I'm not super confident about this name, but the name I'm less confident is the word "active" (part of "ActiveBanner"). 
The idea is that "MaybeBanner" does some business logic and maybe, if it deems so, it imports the banner code. And if that code gets imported, it definitely displays a banner. It used to be that `active-banner.tsx` was smart on itself and figuring out whether to proceed with the *display* of the banner. Now, in this PR, the `active-banner.tsx` is only called if it *definitely* is going to display a banner.

Reply:
I'd be cautious to use „Maybe” then, because it has a [meaning in Functional Programming](https://en.wikipedia.org/wiki/Option_type).

Reply:
> Do you think the name is too whimsical?

Perhaps a little. I am not opposed to it, to be honest. What if we call it just `Banners`?

Reply:
And keep the `ActiveBanner` for the actual banner components?
I can do that.